src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/DwpState.java

```diff
@@ -31,7 +31,8 @@
     PHME_GRANTED("phmeGranted", "PHME granted"),
     PHME_REFUSED("phmeRefused", "PHME refused"),
     DECISION_ACTIONED("decisionActioned", "Decision actioned"),
-    HEARING_RECORDING_PROCESSED("hearingRecordingProcessed","Hearing Recording Processed");
+    HEARING_RECORDING_PROCESSED("hearingRecordingProcessed","Hearing Recording Processed"),
+    HEARING_RECORDING_ACTIONED("hearingRecordingActioned","Hearing Recording Actioned");
```

Comment:
Can we make this lower case like all of the other states? e.g. "Hearing recording actioned" (And also change the one above in 'HEARING_RECORDING_PROCESSED'